.gitignore

```diff
@@ -10,3 +10,6 @@ wolfTPM/api/
 wolfTPM/wolftpm/
 wolfHSM/api
 wolfHSM/docs/xml
+
+# Exclude Visual Studio Debris
+**/.vs
```

Comment:
did you mean `**/*.vs`?

Reply:
No. But good question. There's a Visual Studio `.vs` directory with many files; it is created in not only every project directory, but also any arbitrary directory via "open with Visual Studio".

Reply:
ah ha!